packages/manager-components/src/components/ManagerButton/ManagerButton.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { PropsWithChildren } from 'react';
+import {
+  OsdsButton,
+  OsdsTooltip,
+  OsdsTooltipContent,
+} from '@ovhcloud/ods-components/react';
+import { useTranslation } from 'react-i18next';
+import './translations';
+
+import { useIsAuthorized } from '../../hooks/iam';
+
+export type ManagerButtonProps = PropsWithChildren<{
+  action?: string;
```

Comment:
**nitpick:** I think we can make this `string | Array` to support if the data is formed from the responses of 1 or more API calls.
For example, ` <DATA FROM API 1> - <DATA FROM API 2>`

Reply:
Ok I understand 👍  If the button need more than one action